float32 dt_frame
float32 dt_decompress
float32 camera_f
float32 camera_u0
float32 camera_v0
float32 camera_w
float32 camera_h
